osu.Game.Tests/Resources/TestResources.cs

```diff
@@ -17,14 +18,14 @@ public static class TestResources
 
         public static string GetTestBeatmapForImport(bool virtualTrack = false)
         {
-            var temp = Path.GetTempFileName() + ".osz";
+            var tempPath = Path.Combine(RuntimeInfo.StartupDirectory, Path.GetTempFileName() + ".osz");
```

Comment:
This is documented as returning the full path to the temporary file (https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=netcore-3.1). I don't think we want to insert the startup directory here.

Can you please check to make sure this isn't creating a local subfolder?

Reply:
this is definitely wrong. should be reverted.